addon/-private/instance-initializers/initialize-store-service.js

```diff
@@ -1,12 +1,53 @@
+import { deprecate } from 'ember-data/-private/debug';
 /*
- Configures a registry for use with an Ember-Data
- store.
+  Configures a registry for use with an Ember-Data
+  store.
 
- @method initializeStoreService
- @param {Ember.ApplicationInstance} applicationOrRegistry
- */
+  @method initializeStoreService
+  @param {Ember.ApplicationInstance} applicationOrRegistry
+*/
 export default function initializeStoreService(application) {
   var container = application.lookup ? application : application.container;
   // Eagerly generate the store so defaultStore is populated.
   container.lookup('service:store');
+
+  let initializers = application.application.constructor.initializers;
+  deprecateOldEmberDataInitializers(initializers)
+}
+
+
+
+let deprecatedInitializerNames = ['data-adapter', 'injectStore', 'transforms', 'store'];
```

Comment:
Nevermind. This code is correct. I got `store` confused with the `ember-data` initializer.